scripts/hash-exec-copy.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -e
+
+[[ "$1" =~ https://raw.githubusercontent.com/makerdao/community/* ]] || { echo "Please provide the exec copy link to hash (e.g. url=<https://raw.githubusercontent.com/makerdao/community/...>)"; exit 1; }
+
+for ARGUMENT in "$@"
+do
+    KEY=$(echo "$ARGUMENT" | cut -f1 -d=)
+    VALUE=$(echo "$ARGUMENT" | cut -f2 -d=)
+
+    case "$KEY" in
+            url)      URL="$VALUE" ;;
+            *)        URL="$KEY" 
+    esac
+done
```

Comment:
What's the main reason of this block? To be used when the script is called directly without using `make`?
If that's the case, I think we should tweak it a bit more as, these work:
```
./scripts/hash-exec-copy.sh https://raw.githubusercontent.com/makerdao/community/master/governance/votes/Executive%20Vote%20-%20April%2029%2C%202022.md
./scripts/hash-exec-copy.sh url=https://raw.githubusercontent.com/makerdao/community/master/governance/votes/Executive%20Vote%20-%20April%2029%2C%202022.md
```

which is nice, however if you make a typo in the `url` variable name, like:
```
./scripts/hash-exec-copy.sh uri=https://raw.githubusercontent.com/makerdao/community/master/governance/votes/Executive%20Vote%20-%20April%2029%2C%202022.md
```

It will silently succeed with the hash of empty value.

Reply:
yeah, I guess the same could be for `get-created-timestamp.sh` but in this case it will actually cause an unexpected behaviour, nice catch!

Reply:
it should be addressed in 89705da

Reply:
So now this doesn't work anymore:
```
./scripts/hash-exec-copy.sh https://raw.githubusercontent.com/makerdao/community/master/governance/votes/Executive%20Vote%20-%20April%2029%2C%202022.md
```

Which is better than before, maybe not the optimal.
In any case what I'd definitely add is a message error.

Reply:
yeah, now it strictly enforce the `url` key I guess, I'll see what If there is a better approach.

Reply:
this may fix it d4df582

Reply:
So IMO the best it is to make it work with `url` and without it and fail with an error message with a wrong key.
Second best option is what we have now but adding the missing error message.
Anyway don't waste much time, we will be using it with `make` so to be honest will work the same way.

Reply:
> this might fix it [d4df582](https://github.com/makerdao/spells-goerli/commit/d4df58271dd42f15960af07a977a6bfde4075d2b)

So this now allows to use any key that will work. And it is a bit redundant to do:
```
case "$KEY" in
            url)      URL="$VALUE" ;;
            *)        URL="$VALUE"
    esac
```
If we want this behaviour, you just remove the `case` and do plain `URL="$VALUE"`

Reply:
yeah, not sure having both interoperability would make sense here

Reply:
but maybe better removing the `case` as you suggested for this script as only one key is required

Reply:
it should be done in dfcadb5